drivers/mpu9250/MPU9250.cpp

```diff
@@ -408,31 +321,27 @@ void MPU9250::_measure()
 	}
 
 	if (int_status & BITS_INT_STATUS_FIFO_OVERFLOW) {
-		DF_LOG_ERR("overflow");
 		reset_fifo();
 
 		m_synchronize.lock();
-		++m_sensor_data.fifo_overflow_counter;
+		if ((++m_sensor_data.fifo_overflow_counter % 1000) == 0) {
+			DF_LOG_ERR("FIFO overflow");
```

Comment:
Good point.  This change was a hold over from some debug code I needed when integrating the mag class.  I will delete it.